addon/templates/components/power-select/options.hbs

```diff
@@ -5,18 +5,29 @@
 {{/if}}
 {{#each options as |opt index|}}
   {{#if (ember-power-select-is-group opt)}}
-    <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present opt.disabled}} role="option">
-      <span class="ember-power-select-group-name">{{opt.groupName}}</span>
-      {{#component optionsComponent
-        options=(readonly opt.options)
-        select=(readonly select)
-        groupIndex=(concat groupIndex index ".")
-        optionsComponent=(readonly optionsComponent)
-        role="group"
-        class="ember-power-select-options" as |option|}}
-        {{yield option select}}
-      {{/component}}
-    </li>
+    {{#with
```

Comment:
What about this?

```hbs
  {{#component groupComponent group=opt extra=extra select=select as |option|}}
    {{#component optionsComponent
      options=(readonly opt.options)
      select=(readonly select)
      groupIndex=(concat groupIndex index ".")
      optionsComponent=(readonly optionsComponent)
      role="group"
      class="ember-power-select-options" as |option|}}
      {{yield option select}}
    {{/component}}
  {{/component}}
```

Seems simple enough and users that want to customize the group but not the options can still just `{{yield}}`. Is there any feature I'm missing?